src/components/List/List.tsx

```diff
@@ -0,0 +1,33 @@
+import "./List.scss";
+
+import { FC, useState } from "react";
+
+const selectedItemClassName = "list__item list__item--selected";
+const unselectedItemClassName = "list__item";
+
+interface IGenreListProps {
+    itemNames: string[],
+    preselectedItemName: string,
+    onSelect: (itemName: string) => void,
+}
+
+const List: FC<IGenreListProps> = ({itemNames, preselectedItemName, onSelect}) => {
+    const [selectedItemName, setSelectedItemName] = useState(preselectedItemName);
+    const listItems = itemNames.map((itemName) => {
+        var className = itemName === selectedItemName ? selectedItemClassName : unselectedItemClassName;
```

Comment:
const

Reply:
var replaced to const, thank you for the comment.